Aardvark/ARKEmailBugReporter.m

```diff
@@ -652,3 +620,79 @@ - (BOOL)canBecomeFirstResponder;
 }
 
 @end
+
+
+@implementation ARKDefaultPromptPresenter
+
+- (void)showBugReportingPromptForConfiguration:(nonnull ARKEmailBugReportConfiguration *)configuration completion:(nonnull ARKEmailBugReporterCustomPromptCompletionBlock)completion {
+    self.configuration = configuration;
+    self.completion = completion;
```

Comment:
It might be a little tidier to not store these in properties on the object, but instead capture them in the completion blocks inside the method below (which could then just be inlined here).